lib/record_store/version.rb

```diff
@@ -1,3 +1,3 @@
 module RecordStore
-  VERSION = '4.0.0'
+  VERSION = '4.1.0'.freeze
```

Comment:
The version should be bumped to `4.0.1` since we're following [semantic versioning](http://semver.org/) and this is a bug fix 🙂